### ISSUE TITLE
strongswan: 5.7.X upgrade plans?

### ISSUE THREAD

**Issue by mimugmail** [opnsense/core#2779]
Any timetable or plan to include 5.7.1? I could offer some testing time when you have a pkg ready. 

https://fossies.org/linux/strongswan/NEWS

Not sure if this needs testing:
```
   24 - Dots are not allowed anymore in section names in swanctl.conf and
   25   strongswan.conf. This mainly affects the configuration of file loggers. If the
   26   path for such a log file contains dots it now has to be configured in the new
   27   `path` setting within the arbitrarily renamed subsection in the `filelog`
   28   section.
```

Some changes to UI required (but not necessary:
```
   42 - Support for Postquantum Preshared Keys for IKEv2 (draft-ietf-ipsecme-qr-ikev2)
   43   has been added.
```

**Comment by fichtner** [comment]
5.7.0 is in FreeBSD ports now. It's at least two weeks away for us factoring in build issues with LibreSSL and full testing...

PS: this is a ports ticket. ;)

**Comment by mimugmail** [comment]
Hm, correct (beside the UI part). Just wanted to be sure I can do some testing before release :)

**Comment by fichtner** [comment]
Shipped in 18.7.7.
